test/src/examples_complete_test.go

```diff
@@ -25,8 +25,14 @@ func TestExamplesComplete(t *testing.T) {
 	terraform.InitAndApply(t, terraformOptions)
 
 	// Run `terraform output` to get the value of an output variable
-	outputExample := terraform.Output(t, terraformOptions, "example")
+	outputClusterName := terraform.Output(t, terraformOptions, "cluster_name")
 
 	// Verify we're getting back the outputs we expect
-	assert.Regexp(t, "^Hello world! [0-9]+$", outputExample)
+	assert.Regexp(t, "eg-ue2-test-msk-test-*", outputClusterName)
```

Comment:
My guess is `-*` is the wrong expression here. Do we want to check for a numeric index of the broker?
```suggestion
	assert.Regexp(t, "^eg-ue2-test-msk-test-\d+$", outputClusterName)
```

Reply:
Updated ... using [0-9a-fA-F]+ for the hex part. Also moved to go modules